src/projects/project-create-pull-diff.html

```diff
@@ -0,0 +1,52 @@
+<link rel="import" href="../bower_components/neon-animation/neon-animation-runner-behavior.html">
+<link rel="import" href="../bower_components/neon-animation/animations/fade-in-animation.html">
+<link rel="import" href="../pull-request/pull-request-diff.html">
+<dom-module id="project-create-pull-diff">
+  <template>
+    <style>
+      :host {
+        display: none;
+        background-color: var(--primary-background-color);
+        padding: 2em;
+      }
+      h2 {
+        margin-top: 0;
+      }
+    </style>
+    <h2>Describe your changes</h2>
+    <pull-request-diff editable data-route="diff" diff="[[diff]]"></pull-request-diff>
```

Comment:
I think that would be very confusing for users. And what is the benefit of doing so? We needed to make the diff view editable anyway. Does the place in which it is editable matter from a technical standpoint?

Op 15 mei 2016 om 22:11 heeft Tim van der Lippe <notifications@github.com> het volgende geschreven:

In src/projects/project-create-pull-diff.html:

> +<link rel="import" href="../bower_components/neon-animation/animations/fade-in-animation.html">
> +<link rel="import" href="../pull-request/pull-request-diff.html">
> +<dom-module id="project-create-pull-diff">
> +  <template>
> +    <style>
> +      :host {
> +        display: none;
> +        background-color: var(--primary-background-color);
> +        padding: 2em;
> +      }
> +      h2 {
> +        margin-top: 0;
> +      }
> +    </style>
> +    <h2>Describe your changes</h2>
> +    <pull-request-diff editable data-route="diff" diff="[[diff]]"></pull-request-diff>
This should not be editable right? Only on the actual pull requests pages we want to make them editable?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub